iOS/DuckDuckGo/AIChat/InputBox/SwitchBar/TextEntry/SwitchBarTextEntryView.swift

```diff
@@ -169,7 +169,7 @@ class SwitchBarTextEntryView: UIView {
             textView.returnKeyType = .search
             textView.autocapitalizationType = .none
         case .aiChat:
-            placeholderLabel.text = "Ask Duck.ai..."
+            placeholderLabel.text = "Ask..."
```

Comment:
Created a task for localization https://app.asana.com/1/137249556945/task/1210819617715992?focus=true to not forget about it.